gflownet/gflownet.py

```diff
@@ -191,7 +183,7 @@ def __init__(
         self.jsd = -1.0
 
     def parameters(self):
-        if self.backward_policy.is_model == False:
+        if self.backward_policy.is_model is False:
```

Comment:
Agree with `is` instead of `==` but what is not pythonic about `foo is False` vs `not foo`? `not foo` will be True for things other than `False`, so isn't [explicit better than implicit](https://peps.python.org/pep-0020/)? ^^

Reply:
Hm interesting I'm not longer sure. Since you named the variable `is_model`, which is explicitly boolean, I would personally default to `if is_model` and `if not is_model`, rather than `if is_model is True` and `if is_model is False`, but I take your point.

Reply:
I commented on this [above](https://github.com/alexhernandezgarcia/gflownet/pull/150#discussion_r1277524340), at least that's my take.

Reply:
Yes, you are right about the name of this variable. There are many other cases like this, so I will make these changes another time since it is not urgent. Thanks for comment!